openvdb/Makefile

```diff
@@ -282,6 +284,41 @@ ifneq (2,$(strip $(GLFW_MAJOR_VERSION)))
     CXXFLAGS += -DOPENVDB_USE_GLFW_3
 endif
 
+ifeq (yes,$(strip $(warnings)))
+    USING_CLANG=$(shell ${CXX} -dM -E -x c++ - < /dev/null | grep __clang__)
+    USING_GCC=$(shell ${CXX} -dM -E -x c++ - < /dev/null | grep __GNUC__)
+    ifneq (,$(USING_CLANG))
+        CXXFLAGS += \
+            -Qunused-arguments \
+            -Weverything \
```

Comment:
Evidently, `-Weverything` is [not recommended](https://softwareengineering.stackexchange.com/questions/122608/clang-warning-flags-for-objective-c-development/124574#124574).  I think we've agreed to go with "`-Wall -Wextra -Wconversion`" instead ([OVDB-10](https://jira.aswf.io/browse/OVDB-10)).

Reply:
Sure will do, didn't see that conversation and was going to ask about -Werror!